docs/developer-guide/pprofiling.md

```diff
@@ -0,0 +1,146 @@
+# Performance Profiling in zot
```

Comment:
Instead of assets/profile001.gif 
Convert this into jpg or png

Rename this file and profiling-001.png

Reply:
Rename assets/images/flame.svg and profiling-flame.svg

Reply:
Done